### PR TITLE
Nukepowerup

### PR DESCRIPTION
**Pull request by gerathas**
Finished all the technical parts of the nuke powerup and just need help fixing the button.

### CODE REVIEW

File: Assets/Scripts/GameScreenOverlays/NukeButton.cs [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    public static void activateButton()
+    {
+        nukeButton.interactable = true;
+    }
+
+    public void nukeEffect()
+    {
+        var missiles = GameObject.FindGameObjectsWithTag("Missile");
```
Comment (danielkim10) [inline Assets/Scripts/GameScreenOverlays/NukeButton.cs:27]: Use constants

**Review by danielkim10 — CHANGES_REQUESTED** [review]
- Nuke button shouldn't overlap player
- When button is interactable icon doesn't change color, it still uses the grey version
- If you zoom in close you can see the word Button on the nuke button, please remove

File: Assets/Scripts/GameScreenOverlays/NukeButton.cs (hunk 1 above)
Comment (gerathas) [reply Assets/Scripts/GameScreenOverlays/NukeButton.cs]: Fixed these issues

File: Assets/Scripts/Entities/PlayerMovement.cs [hunk 2]
```diff
@@ -78,5 +78,11 @@ private void OnTriggerEnter2D(Collider2D collision)
         {
             hp += 1;
         }
+
+        if (collision.tag == Constants.NUKEPOWERUP_TAG)
+        {
+            NukeButton script = GameObject.Find("NukeButton").GetComponent<NukeButton>();
```
Comment (danielkim10) [inline Assets/Scripts/Entities/PlayerMovement.cs:84]: Constants !!

**Review by danielkim10 — CHANGES_REQUESTED** [review]
- Button still overlaps player, refer to the setSiblingIndex function
- Button will show as active but nothing happens when I click on it

**Review by Raymond-Xia — APPROVED** [review]

**Review by danielkim10 — APPROVED** [review]

**Pull request merged**